src/app/shared/data-input/boolean/boolean-data-input.component.scss

```diff
@@ -8,3 +8,8 @@ div.custom-control {
   height: 1.5rem;
   padding-left: 2rem !important;
 }
+
+.custom-control-input {
+  top: 0;
+  right: 0;
```

Comment:
Does it work correctly across all perspectives?

Reply:
Yes.